packages/renderer/src/stores/providers.ts

```diff
@@ -37,10 +37,10 @@ export async function fetchProviders() {
 export const providerInfos: Writable<ProviderInfo[]> = writable([]);
 
 // need to refresh when extension is started or stopped
-window.addEventListener('extension-started', () => {
+window?.events?.receive('extension-started', () => {
```

Comment:
I see somewhere it is `window?`, somewhere else `window`. Not sure if window can be undefined but if this happens it would be better to use `window?` in all updates